modules/ivc_champva/app/controllers/ivc_champva/v1/uploads_controller.rb

```diff
@@ -79,19 +79,37 @@ def supporting_document_ids(parsed_form_data)
           parsed_form_data['supporting_docs']&.pluck('claim_id')&.compact.presence || []
       end
 
+      # rubocop:disable Metrics/MethodLength
+      # rubocop:disable Style/IdenticalConditionalBranches
```

Comment:
We generally don't like disabling rubocops, but since this is related to a Flipper, I'll let it slide 😄 since it'll change once the flipper is removed completely.